projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/notification/EmailPropertiesConfiguration.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2021-2023 VMware, Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.vmware.taurus.service.notification;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class EmailPropertiesConfiguration {
+
+  private final String MAIL_SMTP_PASSWORD = "mail.smtp.password";
+  private final String MAIL_SMTP_USER = "mail.smtp.user";
+  private final String MAIL_SMTP_AUTH = "mail.smtp.auth";
+  private final String TRANSPORT_PROTOCOL = "transport.protocol";
+
+  @Bean
+  @ConfigurationProperties(prefix = "mail")
+  Map<String, String> mail() {
+    return new HashMap<>();
+  }
+
+  public Map<String, String> smtpWithPrefix() {
+    Map<String, String> result = new HashMap<>();
+    var props = this.mail();
+    props.forEach(
+        ((key, value) -> {
+          if (key.startsWith("smtp")) {
+            result.put("mail." + key, value);
+          }
+        }));
+    return result;
+  }
+
+  public String getUsername() {
+    return smtpWithPrefix().get(MAIL_SMTP_USER);
+  }
+
+  public String getPassword() {
+    return smtpWithPrefix().get(MAIL_SMTP_PASSWORD);
+  }
+
+  public boolean isAuthEnabled() {
+    return Boolean.parseBoolean(smtpWithPrefix().get(MAIL_SMTP_AUTH));
+  }
+
+  public String getTransportProtocol() {
```

Comment:
add short description,  what is "transport protocol" and what are expected values for transport protocol ?